contents/blog/feature-flags-vs-configuration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Feature flags vs configuration: what should you choose?"
+date: 2023-06-27
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Feature flags
+ - Explainers
+---
+
+Sometimes you want to change the behavior of your app without changing the code. Both feature flags and configurations are options for doing this. 
+
+In their most basic form, they are a key you add to your code that returns a value you can control. You can use this value to change the behavior or experience of your app or product.
+
+This post goes over what configuration and feature flags do, and which one you should choose.
+
+## What is a configuration?
+
+A configuration, or config, is literally the definition above. A key when called with a service returns a value. You set both the key and value, then add the key and service to your code. You can then modify the key’s value. Configurations are often separated into a dedicated file or even handled by an external service. 
```

Comment:
"is literally the definition above" didn't help me understand it.

What is a "service" in this context? A library? Framework? something else? Let's clarify it

```suggestion
A configuration enables you to call a key with a service and return a specific value. You set both the key and value, and then add the key and service to your code. You can then modify the key’s value. Configurations are often separated into a dedicated file or even handled by an external service. 
```

Reply:
Rewrote to clarify